mks/service/mks_client_bsn_hr.py

```diff
@@ -141,7 +141,8 @@ def extract_for_bsn(xml_data):
         is_amsterdammer = False
         for i in vestigingen:
```

Comment:
`i`?

Reply:
Kan er `v` van maken. `i` gebruik ik vaker in dit soort context.